corehq/apps/style/templates/style/includes/modal_report_issue.html

```diff
@@ -72,9 +72,23 @@ <h4 class="modal-title">{% trans "Report an Issue with CommCare HQ" %}</h4>
                         </div>
                         {% endif %}
 
+                        {% if request.couch_user.is_domain_admin %}
```

Comment:
hmm i do see that now, when i first read it i thought it was ensuring that they did not fill have to fill it out twice. i feel like that's a confusing UX to have a box appear for project description once, and then never again. there's no reason anyone would know that this corresponds to the box in the project settings if they ever wanted to change it